app/models/user.rb

```diff
@@ -6,10 +6,11 @@ class User < ActiveRecord::Base
          :recoverable, :rememberable, :trackable, :validatable
 
   validates :display_name,
-  presence: true,
-  length: { maximum: 50 }
+    presence: true,
```

Comment:
Okay, it looks like when we push our code our spacing disappears, because we *definitely* had those spaces in there before we pushed. Is it ok if we don't fix this?